### PR TITLE
(feat)docs: Added docs for openebs experiments

### PR DESCRIPTION
**Pull request by somesh2905**
Signed-off-by: somesh kumar <somesh.kumar@mayadata.io>
1. Added docs for openebs experiments
- openebs-target-network-loss
- openebs-pool-pod-failure
- openebs-pool-container-failure
2. Resolved typo in openebs-target-network-delay docs

### CODE REVIEW

File: docs/openebs-pool-container-failure.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+id: openebs-pool-container-failure
```
Comment (ispeakc0de) [inline docs/openebs-pool-container-failure.md]: For containers, we are following container kill convention. Can you please change it to `openebs-pool-container-kill`

File: docs/openebs-pool-container-failure.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+title: OpenEBS Pool Container Failure Experiment Details
+sidebar_label: OpenEBS Pool Container Failure
```
Comment (ispeakc0de) [inline docs/openebs-pool-container-failure.md]: same as above

File: docs/openebs-pool-container-failure.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- Follow the steps in the sections below to prepare the ChaosEngine & execute the experiment.
+
+### Prepare ChaosEngine
+
+- Provide the application info in `spec.appinfo`
+- Override the experiment tunables if desired
+
+#### Supported Experiment Tunables
+
+| Variables             | Description                                                  | Type      | Notes                                                      |
```
Comment (ispeakc0de) [inline docs/openebs-pool-container-failure.md:76]: Can you please add lib image env variable?

File: docs/openebs-pool-container-failure.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+## Details
+
+- This scenario validates the behaviour of stateful applications and OpenEBS data plane upon forced termination of the target pod container
+- Containers are killed using the kill command provided by pumba
+- Pumba is run as a daemonset on all nodes in dry-run mode to begin with; the kill command is issued during experiment execution via kubectl exec
+- Can test the stateful application's resilience to momentary iSCSI connection loss
+
+## Integrations
+
+- Container kill is achieved using the pumba chaos library in case of docker runtime, & litmuslib using crictl tool in case of containerd runtime.
```
Comment (ispeakc0de) [inline docs/openebs-pool-container-failure.md]: Is this experiment support containerd runtime?

File: docs/openebs-pool-container-failure.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    applabel: 'app=percona'
+    appkind: deployment
+  chaosServiceAccount: percona-sa
+  monitoring: false
+  jobCleanUpPolicy: delete
+  experiments:
+    - name: openebs-pool-container-failure
+      spec:
+        components:
+          - name: FORCE
```
Comment (ispeakc0de) [inline docs/openebs-pool-container-failure.md]: is this env required?

File: docs/openebs-target-network-loss.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+## Experiment Metadata
+
+| Type      | Description              | Tested K8s Platform                                               |
+| ----------| ------------------------ | ------------------------------------------------------------------|
+| OpenEBS   | Induce network loss into the cStor target/Jiva controller container | GKE, Konvoy(AWS), Packet(Kubeadm), OpenShift(Baremetal)  |
+
+## Prerequisites
+
+- Ensure that the Kubernetes Cluster uses Docker runtime
+- Ensure that the Litmus Chaos Operator is running
```
Comment (ispeakc0de) [inline docs/openebs-target-network-loss.md]: Can you please provide the link of operator.yaml https://github.com/litmuschaos/chaos-operator/blob/master/deploy/operator.yaml

File: docs/openebs-target-network-loss.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+- Provide the application info in `spec.appinfo`
+- Override the experiment tunables if desired
+
+#### Supported Experiment Tunables
+
+| Variables             | Description                                                  | Type      | Notes                                                      |
+| ----------------------| ------------------------------------------------------------ |-----------|------------------------------------------------------------|
+| APP_PVC               | The PersistentVolumeClaim used by the stateful application   | Mandatory | PVC may use either OpenEBS Jiva/cStor storage class        |
+| DEPLOY_TYPE           | Type of Kubernetes resource used by the stateful application | Optional  | Defaults to `deployment`. Supported: `deployment`, `statefulset`|
+| CONTAINER_RUNTIME     | The container runtime used in the Kubernetes Cluster         | Optional  | Defaults to `docker`. Supported: `docker`                  |
```
Comment (ispeakc0de) [inline docs/openebs-target-network-loss.md]: We don't have container_runtime env in job.yaml

File: docs/openebs-target-network-loss.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+| LIB_IMAGE             | The chaos library image used to inject the network loss           | Optional  | Defaults to `gaiaadm/pumba:0.4.8`. Supported: `gaiaadm/pumba:0.4.8`|                
+| TARGET_CONTAINER      | The container into which delays are injected in the storage controller pod  | Optional  | Defaults to `cstor-istgt`                   |
```
Comment (ispeakc0de) [inline docs/openebs-target-network-loss.md]: this field is also not present inside job

File: docs/openebs-target-network-loss.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+- Follow the steps in the sections below to prepare the ChaosEngine & execute the experiment.
+
+### Prepare ChaosEngine
+
+- Provide the application info in `spec.appinfo`
+- Override the experiment tunables if desired
+
+#### Supported Experiment Tunables
+
+| Variables             | Description                                                  | Type      | Notes                                                      |
```
Comment (ispeakc0de) [inline docs/openebs-target-network-loss.md:76]: please add NETWORK_PACKET_LOSS_PERCENTAGE env variable here

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

File: docs/openebs-pool-container-failure.md (hunk 1 above)
Comment (somesh2905) [reply docs/openebs-pool-container-failure.md]: @shubhamchaudhary  Chaosexperiment cr is getting created with this name only. So, should I change or not

File: docs/openebs-pool-container-failure.md [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+- OpenEBS Storage target pods are healthy
+
+If the experiment tunable DATA_PERSISTENCE is set to 'enabled':
+
+- Application data written prior to chaos is successfully retrieved/read 
+- Database consistency is maintained as per db integrity check utils 
+
+## Details
+
+- This scenario validates the behaviour of stateful applications and OpenEBS data plane upon forced termination of the target pod container
```
Comment (w3aman) [inline docs/openebs-pool-container-failure.md]: use targeted pool pod container instead of target pod container

**Review by w3aman — CHANGES_REQUESTED** [review]

File: docs/openebs-pool-container-failure.md (hunk 1 above)
Comment (somesh2905) [reply docs/openebs-pool-container-failure.md]: modified

File: docs/openebs-pool-container-failure.md (hunk 3 above)
Comment (somesh2905) [reply docs/openebs-pool-container-failure.md:76]: added

File: docs/openebs-pool-container-failure.md (hunk 4 above)
Comment (somesh2905) [reply docs/openebs-pool-container-failure.md]: no, removed

File: docs/openebs-target-network-loss.md (hunk 6 above)
Comment (somesh2905) [reply docs/openebs-target-network-loss.md]: added

**Review by w3aman — DISMISSED** [review]
/lgtm

**Review by ispeakc0de — DISMISSED** [review]

File: docs/openebs-pool-container-failure.md [hunk 11]
```diff
@@ -0,0 +1,131 @@
...
+
+- Application pods are healthy before chaos injection
+- Application writes are successful on OpenEBS PVs
+
+## Exit Criteria
+
+- Stateful application pods are healthy post chaos injection
+- OpenEBS Storage target pods are healthy
+
+If the experiment tunable DATA_PERSISTENCE is set to 'enabled':
```
Comment (imrajdas) [inline docs/openebs-pool-container-failure.md:46]: use 

> `DATA_PERSISTENCE`

**Review by ajeshbaby — APPROVED** [review]
LGTM

**Review by ksatchit — APPROVED** [review]

**Pull request merged**